src/com/javabasic/lesson14/homework/buildings/MainBuildings.java

```diff
@@ -0,0 +1,14 @@
+package com.javabasic.lesson14.homework.buildings;
+
+public class MainBuildings {
+    public static void main(String[] args) {
+        IndustrialBuildings bld1 = new IndustrialBuildings();
+        ResidentialBuildings bld2 = new ResidentialBuildings(10);
+        CommercialBuildings bld3 = new CommercialBuildings(99);
+
+        System.out.println(bld1.calcCost(100 , 2));
```

Comment:
area and tax rate need to setup as fields for the ...Building class, they are state of building the same as income, subsidies.

And calculate method become simplier